Assets/BossRoom/Scripts/Client/Game/Character/ClientCharacterVisualization.cs

```diff
@@ -65,6 +65,13 @@ public class ClientCharacterVisualization : NetworkBehaviour
 
         event Action Destroyed;
 
+
+        /// <summary>
+        /// When true, the visualization moves itself to keep in sync with the network transform.
+        /// (This can be disabled by special actions that want to reposition the visualization manually.)
+        /// </summary>
+        bool m_IsFollowingNetworkTransform = true;
```

Comment:
Why the name "network transform"? I thought we weren't using MLAPI's NetworkTransform yet? (let's make sure we don't confuse our users and use different words than this in all of this PR)

Reply:
this variable and all related code was removed (by request of another comment) so issue is moot!